tests/test_main.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 import unittest
 
-from language_detector import detect_language
+from language_detector import detect_language, LANGUAGES
```

Comment:
Yeah it's being used in half the tests. In the other half self.languages is being used instead.